Draft for adding services for dev environment

ValterAro opened the pull request:
Added 10 services with the weather service still needing work (weather service for created for ruuter one and will need some work to convert the login in the json files into yml as well as datamapper changes). Will also add E-Business register services later on. Also created service-post.sql file in the DSL/Resql folder and added a initiate-service.yml file to initiate the necessary service.

ValterAro committed:
Draft for adding services for dev environment
Added 10 services with the weather service still needing work. Will also add E-Business register services later on. Also created service-post.sql file in the DSL/Resql folder and added a initiate-service.yml file to initiate the necessary service.

ValterAro committed:
Added a service to get the most popular public initiatives
In addition to the public initiative service, made some small fixes to the urls in some cases.

1AhmedYasser commented on DSL/Ruuter.public/DSL/GET/labour-market-headline-indicators.yml:
```diff
@@ -0,0 +1,70 @@
+    year: 2023
+    month: ${new Date().getMonth()}
+    divided: ${(month + 3) / 3}
+    #    quarter: ${String(divided)[0]}
```
Remove comments

ValterAro replied:
Removed comments

1AhmedYasser commented on DSL/Ruuter.public/DSL/POST/weather-service.yml:
```diff
@@ -0,0 +1,5 @@
+  return: "Ilm on hetkel ilus ja päike paistab."
+  next: end
+
+# Old weather service needs to be converted to yml from json in this file
```
Is this in-complete ?

ValterAro replied:
Deleted the file

ValterAro committed:
Update labour-market-headline-indicators.yml

Removed comments.

ValterAro committed:
Delete DSL/Ruuter.public/DSL/POST/weather-service.yml

Deleted the weather service file.

ValterAro committed:
Update intiate-service.yml

Removed references to the weather service as it is currently not functional.

ValterAro committed:
Added some corrections to the initiative events service
Improved the service that shows five of the most popular currently ongoing public initiatives.
